linux/root_test.go

```diff
@@ -194,10 +191,41 @@ func TestRootLVMExtend(t *testing.T) {
 	foundLv := vgs[vgname].Volumes[lvname]
 	ast.Equalf(size1, foundLv.Size, "initial volume size incorrect")
 
+	mount1 := path.Join(tmpDir, "mp1")
+	os.Mkdir(mount1, 0755)
+
+	if err := runCommand("mkfs.ext4", "-F", "-L"+lvname, foundLv.Path); err != nil {
+		t.Errorf("Failed to mkfs on %s: %s", foundLv.Path, err)
+	}
+
+	if err := runCommand("mount", foundLv.Path, mount1); err != nil {
+		t.Errorf("Failed mount: %s", err)
+	}
+
+	cl.AddF(func() error { return runCommand("umount", mount1) }, "unmount lv1")
```

Comment:
Mostly for my own edification, but is there a reason to exec the mount/umount commands here vs. using the syscalls?

Reply:
I changed to unix.Mount and unix.Unmount .